### ISSUE TITLE
update_run.sh is created as a directory and not script, when docker-compose downloads it.

### ISSUE THREAD

**Issue by upenbendre** [provectus/kafka-ui#3010]
**Describe the bug**
Using https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui.yaml , only the kafka-ui container starts, while others exit. The Kafka containers throw the error:
ERROR: Did you forget the update_run.sh file that came with this docker-compose.yml file?

**Set up**
On a MacOS with Docker Desktop , or on a CentOS 7 machine, download the docker-compose from here https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui.yaml 

**Steps to Reproduce**
Run docker-compose up -d --build
check the scripts directory. the file update_run.sh is created as a directory instead of a file.

**Expected behavior**
scripts/update_run.sh should be created as a directory

**Screenshots**

**Additional context**

**Comment by Haarolean** [comment]
Hey, thanks for reaching out. That's a docker daemon behavior, if the volume bind source file doesn't exist, it will create an empty directory instead. You had to download the sh file alongside with it. We'll see what we can do to make this easier.

**Comment by ashishamway** [comment]
I am facing this issue. When I ran below command, after 5-6 sec, just kafka-ui container continue to run while other container fail. It mainly happen due kafka0 and kafka1 container. When I checked logs it says

>  bash: /tmp/update_run.sh: Permission denied

Command I ran is given below. for that I extract out just _kafka-ui.yaml_ and _scripts_ folder having just one file _update_run.sh_ into an empty folder :
> docker-compose -f ./kafka-ui.yaml up -d

**Comment by Haarolean** [comment]
@ashishamway `chmod u+x update_run.sh` as a workaround

**Comment by brandon3123** [comment]
Did this ever get resolved? I'm getting the same error. 👀  Where is this `update_run.sh` script?

**Comment by Haarolean** [comment]
@brandon3123 nope, as I don't know how this could be possibly solved. `update_run.sh` is a file that lies near the compose file itself, which you need to curl as well.
